### DISCUSSION TITLE
Can box86 run inside ESP32?

### DISCUSSION THREAD

**Discussion by maxbrito500** [ptitSeb/box86#780] [General]
This is certainly a silly question.. but begs to be asked hoping for a non-zero possibility to happen.

Can a cheap 5 USD device powered by an ESP32 CPU (Xtensa LX7) run box86? 

Thanks in advance!

**Comment by ptitSeb** [comment]
Without dynarec, and if you have your ESP32 running linux, then yeah, it should work? But no dynarec means slow performances.

**Comment by maxbrito500** [comment]
OK, thanks for the quick answer. So I guess it won't happen so soon. :-(
